packages/core/useAdBlockerDetector/index.ts

```diff
@@ -0,0 +1,23 @@
+import type { Ref } from 'vue-demi'
+import { ref } from 'vue-demi'
+import { defaultWindow } from '../_configurable'
+
+// Parameters taken from easylist general advert blocking filters
+// https://easylist.to/easylist/easylist.txt
+const HONEY_POT_URL = '/&ad_code=ad&ad_height=ad&ad_ids=ad&ad_network_ad&ad_slot=ad&ad_sub=ad&ad_system=ad'
+
+const hasAdBlocker = ref(false)
```

Comment:
should be placed inside of the function

Reply:
I'm not sure about that because there is a possibility to check for ad blocker more than once, in that case i don't think we should define `hasAdBlocker` again, instead just update it's value every time we call `useAdBlockerDetector`